src/OpenSpaceNet.cpp

```diff
@@ -165,18 +165,36 @@ void OpenSpaceNet::initLocalImage()
 {
     OSN_LOG(info) << "Opening image..." ;
     image_ = make_unique<GdalImage>(args_.image);
-    DG_CHECK(!image_->spatialReference().isLocal(), "Input image is not geo-registered");
 
     bbox_ = cv::Rect{ { 0, 0 }, image_->size() };
+    bool ignoreArgsBbox = false;
+
+    TransformationChain llToPixel;
+    if (!image_->spatialReference().isLocal()) {
+        llToPixel = {
+                image_->spatialReference().fromLatLon(),
+                image_->pixelToProj().inverse()
+        };
+        sr_ = SpatialReference::WGS84;
+    } else {
+        OSN_LOG(warning) << "Image has geometric metadata which cannot be converted to WGS84.  "
```

Comment:
No need to repeat `<<`, just use the same string. The only time you need additional `<<` is to add differently typed arguments. One of the problems with doing this stream-style syntax for logging is that sometimes it can get out-of-sync between threads, which can be a pain, so minimizing the number of stream writes is worthwhile. C++ allows you to break strings between lines. This is the way this warning and the one below it should look like.
```cpp
     OSN_LOG(warning) << "Image has geometric metadata which cannot be converted to WGS84.  "
                                            "Output will be in native space, and some output formats will fail.";
```